runtime/moonbeam/src/lib.rs

```diff
@@ -840,6 +840,103 @@ impl Default for ProxyType {
 	}
 }
 
+fn is_governance_precompile(precompile_name: &precompiles::PrecompileName) -> bool {
+	matches!(
+		precompile_name,
+		PrecompileName::DemocracyPrecompile
+			| PrecompileName::CouncilInstance
+			| PrecompileName::TechCommitteeInstance
+			| PrecompileName::TreasuryCouncilInstance
+			| PrecompileName::PreimagePrecompile
```

Comment:
missing opengov related porecompiles: Referenda, ConvictionVoting and OpenTechCommittee

Reply:
Those are not available on moonbeam yet, I added them as a comment.